packages/ag-charts-community/src/scene/changeDetectable.ts

```diff
@@ -89,7 +89,7 @@ function buildCheckDirtyChain(setterFn: Function, opts: SceneChangeDetectionOpti
         return function (this: any, value: any) {
             const change = setterFn.call(this, value);
 
-            if (change !== NO_CHANGE && value != null && value._dirty) {
+            if (change !== NO_CHANGE && value?._dirty) {
```

Comment:
Just because this is a hot path -

```suggestion
            if (change !== NO_CHANGE && value?._dirty === true) {
```

Means the engine can do fewer checks